astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2685,6 +2713,7 @@ def test_merge_arrays_str(self, flatten):
 all_wrapped_functions = get_wrapped_functions(
     np, np.fft, np.linalg, np.lib.recfunctions
 )
+all_wrapped_functions |= SUPPORTED_NEP35_FUNCTIONS
```

Comment:
Can we add a comment referring to your numpy issue? Hopefully, we won't need to use `SUPPORTED_NEP35_FUNCTIONS` here eventually.